marquez/airflow.py

```diff
@@ -1,93 +1,84 @@
 import json
 import pendulum
-import airflow.models
+from airflow.models import DAG, Log
 from airflow.utils.db import provide_session
 from marquez_client.marquez import MarquezClient
 
+from marquez.utils import JobIdMapping
 
-class MarquezDag(airflow.models.DAG):
+
+class MarquezDag(DAG):
+    _jobIdMapping = None
```

Comment:
Does `_jobIdMapping` define a mapping between a `marquez` job and `airflow` job?

Reply:
yes, and in this case is implemented using airflow Variables.

Reply:
yes, and currently is implemented using airflow Variables.

Reply:
minor: you may want to consider a more expressive variable name then? ex: `_marquez_to_airflow_job_ids`